pkg_resources/_vendor/pyparsing.py

```diff
@@ -3162,6 +3233,8 @@ def z(*paArgs):
 #
 # global helpers
 #
+

```

Comment:
Here's a case where maybe the extra space makes sense. The comment isn't pertinent to the immediate following block but to the rest of the file.